ClavaWeaver/resources/clava/test/weaver/ExprStmt.js

```diff
@@ -0,0 +1,8 @@
+laraImport("weaver.Query")
+
+for(const $loop of Query.search("function", "loopExprStmt").search("loop")) {
+	println(`Loop ${$loop.line} cond: ${$loop.cond.code}`)
+	if($loop.step !== undefined) {
+		println(`Loop ${$loop.line} step: ${$loop.step.code}`)		
+	}
+}
```

Comment:
nitpick: add newline at end of file :)

Reply:
generally speaking, you might want to run a code formatter like prettier to keep code styling consistent